src/Sulu/Bundle/TranslateBundle/Tests/Functional/Translate/ExportTest.php

```diff
@@ -41,279 +32,88 @@ public static function setUpBeforeClass()
-        $translation2->setCode($code2);
-        $translation2->setValue('Exports are great');
-        $this->em->persist($translation2);
-
-        $translation3 = new Translation();
-        $translation3->setCatalogue($catalogue);
-        $translation3->setCode($code3);
-        $translation3->setValue('Exports are configurable');
-        $this->em->persist($translation3);
-
-        //
-        // Package - id 2
-        // -------------------------------
-
-        //Insert some data in the database
-        $package2 = new Package();
-        $package2->setName('Export2');
-        $this->em->persist($package2);
-
-        $catalogue2 = new Catalogue();
-        $catalogue2->setPackage($package2);
-        $catalogue2->setIsDefault(false);
-        $catalogue2->setLocale('en');
-        $this->em->persist($catalogue2);
-
-        $location21 = new Location();
-        $location21->setName('Newsletter');
-        $location21->setPackage($package2);
-        $this->em->persist($location21);
-
-        $location22 = new Location();
-        $location22->setName('Portals');
-        $location22->setPackage($package2);
-        $this->em->persist($location22);
-
-        $code21 = new Code();
-        $code21->setPackage($package2);
-        $code21->setCode('export.easy2');
-        $code21->setBackend(true);
-        $code21->setFrontend(true);
-        $code21->setLocation($location21);
-        $this->em->persist($code21);
-
-        $code22 = new Code();
-        $code22->setPackage($package2);
-        $code22->setCode('export.great2');
-        $code22->setBackend(true);
-        $code22->setFrontend(false);
-        $code22->setLocation($location21);
-        $this->em->persist($code22);
-
-        $code23 = new Code();
-        $code23->setPackage($package2);
-        $code23->setCode('export.configurable2');
-        $code23->setBackend(false);
-        $code23->setFrontend(true);
-        $code23->setLocation($location22);
-        $this->em->persist($code23);
-
-        $this->em->flush();
-
-        $translation21 = new Translation();
-        $translation21->setCatalogue($catalogue2);
-        $translation21->setCode($code21);
-        $translation21->setValue('Exports made super easy');
-        $this->em->persist($translation21);
-
-        $translation22 = new Translation();
-        $translation22->setCatalogue($catalogue2);
-        $translation22->setCode($code22);
-        $translation22->setValue('Exports are super great');
-        $this->em->persist($translation22);
-
-        $translation23 = new Translation();
-        $translation23->setCatalogue($catalogue2);
-        $translation23->setCode($code23);
-        $translation23->setValue('Exports are super configurable');
-        $this->em->persist($translation23);
-
-        $this->em->flush();
+        /** @var \Symfony\Component\Console\Output\OutputInterface $output */
+        $output = $this->createMock('Symfony\Component\Console\Output\OutputInterface');
```

Comment:
Please use [Prophecy](https://github.com/phpspec/prophecy) for the tests. It's included in PHPUnit.